src/utils/dates.ts

```diff
@@ -10,3 +10,13 @@ const ONE_DAY_IN_MILLISECONDS = 86400000;
 export function getFutureDate(offset: number) {
 	return new Date(Date.now() + offset * ONE_DAY_IN_MILLISECONDS);
 }
+
+export function is24HoursLater(purchaseDate: Date): boolean {
+	const currentTime = new Date();
+
+	const timeElapsedInMilliseconds = Math.abs(
+		currentTime.getTime() - purchaseDate.getTime(),
+	);
+
+	return timeElapsedInMilliseconds >= ONE_DAY_IN_MILLISECONDS;
+}
```

Comment:
Praise: The use of is24HoursLater to control the checkbox state for 24 hours after purchase is clear and intuitive. It directly addresses the requirements.